arches_her/templates/views/components/reports/consultation.htm

```diff
@@ -47,7 +44,627 @@ <h1 class="aher-report-toolbar-title"><span class="aher-report-name">{% trans "C
+                                            <tr>
+                                                <td data-bind="text: reference"></td>
+                                                <td data-bind="text: referenceType"></td>
+                                                <td data-bind="text: agency"></td>
+                                                <td class="aher-table-control">
+                                                    <div data-bind="if: $parent.cards['external system references']">
+                                                        <a data-bind="click: function() {$parent.editTile(tileid, $parent.cards['external system references'])}">
+                                                            <i class="fa fa-mail-reply"></i>
+                                                        </a>
+                                                        <a data-bind="click:  function() {$parent.deleteTile(tileid, $parent.cards['external system references'])}">
+                                                            <i class="fa fa-trash"></i>
+                                                        </a>
+                                                    </div>
+                                                </td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- /ko -->
+                    </div>
+                </div>
+            </div>
+            <!-- Contacts Tab -->
+            <div class="aher-report-page" data-bind="if: activeSection() === 'contacts'">
+                <h2 class="aher-report-section-title"><i data-bind="click: function() {toggleVisibility(visible.contacts)}, css: {'fa-angle-double-right': visible.contacts(), 'fa-angle-double-up': !visible.contacts()}" class="fa toggle"></i> {% trans "Contacts" %}</h2>
+                <a data-bind="{if: cards.contacts, click: function(){ addNewTile(cards.contacts) }}" class="aher-report-a">
+                    <i class="fa fa-mail-reply"></i>
+                    <span data-bind="if: cards.contacts.tiles().length">{% trans "Edit Contacts" %}</span>
+                    <span data-bind="ifnot: cards.contacts.tiles().length">{% trans "Add Contacts" %}</span>
+                </a>
+
+                <!-- Collapsible content -->
+                <div data-bind="visible: visible.contacts" class="aher-report-collapsible-container pad-lft">
+                    <!-- ko ifnot: contacts() -->
+                    <div class="aher-nodata-note">No contact for this consultation</div>
+                    <!-- /ko -->
+
+                    <!-- ko if: contacts() -->
+                    <div class="aher-report-subsection">
+
+                        <div class="firstchild-container">
```

Comment:
This might be better served with the keyvalue scene that already handles this kind of data - but these should be resource links.  Also there is no consulting contact card visible in the editor, so this will always be blank.

Reply:
Some of the nodes are the resource-list datatype so the nodes are updated to represented as tables.